src/config/dynamic-theme-fixes.config

```diff
@@ -19885,6 +19885,22 @@ INVERT
 .hero-cover
 .hero-cover p
 
+================================
+
+vjudge.net
+
+CSS
+body {
+    background: none !important;
+}
+#contest-rank-table td.accepted {
+    background-color: rgb(0, 96, 0);
+}
+#contest-rank-table td.prob.fb {
+    background-color: rgb(0, 72, 0);
+}
```

Comment:
These are cosmetic fixes, please remove them.

Reply:
The "darkness" of these two elements are not supposed to be inverted. The convention for most online judges (including ICPC) is darker = earlier. Should I use `INVERT` instead or would that also be considered a cosmetic fix?

Reply:
> The "darkness" of these two elements are not supposed to be inverted. Please see the unaltered scoreboard screenshot for an example of what they should look like before Dark Reader inverts them.

I understand the green one to be fixed, not sure about the red one to be "less noticeable". That really is on the line of cosmetic fixes for me.



> Maybe there's a way I could force Dark Reader to not invert the colors of these two elements without resorting to CSS fixes?

Unfortunately not, you could use:

```css
#contest-rank-table td.accepted {
    background-color: ${lightgreen} !important;
}
#contest-rank-table td.prob.fb {
    background-color: ${darkred} !important;
}

Reply:
I didn't change the reds, both CSS overrides are for the green elements. `td.prob.fb` needs to be a darker than `td.accepted`. I guess I'd just need to use `${lightgreen}` and `${darkgreen}` instead of hardcoding the RGB values?

Reply:
>  I guess I'd just need to use ${lightgreen} and ${darkgreen} instead of hardcoding the RGB values?

Yes

Reply:
Hmm, `${lightgreen}` results in poor contrast with the timestamp and # of tries, is there a list of other colors I can try?

![image](https://user-images.githubusercontent.com/43037910/187213173-01c87b85-d1d5-484c-8722-547c7398ca49.png)

Reply:
Perhaps this could work? They're sort of "hardcoded" though, if that's what you're trying to avoid.

```css
#contest-rank-table td.accepted {
    background-color: ${#b8ffb8} !important;
}
#contest-rank-table td.prob.fb {
    background-color: ${#c8ffc8} !important;
}
```

![image](https://user-images.githubusercontent.com/43037910/187220386-0f5dc78b-ffd1-4a05-ae98-34b81129e600.png)

Reply:
> Perhaps this could work? They're sort of "hardcoded" though, if that's what you're trying to avoid.

If it's using `${...}` it should be fine.

Reply:
Alright, committed. Thank you for the review!